chainerx_cc/chainerx/backward_builder.cc

```diff
@@ -159,7 +159,7 @@ std::vector<RetainedInputToken> BackwardBuilder::RetainInput(std::vector<size_t>
     for (size_t i : indices) {
         CHAINERX_ASSERT(i < inputs_.size());
         input_retention_record_.Record(i);
-        token.emplace_back((RetainedInputToken){internal::GetArrayBody(gsl::at(inputs_, i))->GetParams(), i});
+        token.emplace_back(RetainedInputToken(internal::GetArrayBody(gsl::at(inputs_, i))->GetParams(), i));
```

Comment:
Since we're using `emplace_back`, how about simply passing comma separated ctor arguments?
```suggestion
        token.emplace_back(internal::GetArrayBody(gsl::at(inputs_, i))->GetParams(), i);
```